### PR TITLE
Logical fallacies

### PR DESCRIPTION
**Pull request by jkugs**

### CODE REVIEW

**Comment by gemarcano** [comment]
This addresses issue #101, #87 (are those two issues the same?)

**Comment by jkugs** [comment]
Yes those look the same

File: src/logic_checker.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	def checkNumFieldLogic(field, decimal)
+		absoluteMin = Object.const_get(field.type.upcase + "_MIN")
+		absoluteMax = Object.const_get(field.type.upcase + "_MAX")
+		
+		!decimal ? value = field.value.chardata[0].to_i : value = field.value.chardata[0].to_f
+		
+		if defined? field.min and not field.min.empty?
+			!decimal ? min = field.min.chardata[0].strip.to_i : min = field.min.chardata[0].strip.to_f
+			if min < absoluteMin or value < min
+				puts "\nERROR: check min #{field.title.chardata[0].strip} value in XML"
```
Comment (gemarcano) [inline src/logic_checker.rb]: Would it be possible to make the error messages more verbose/clearer? For example:
`ERROR: check "min" (25) and "value" (20) in "A number", values are inconsistent!`

**Comment by gemarcano** [comment]
It looks fine other than the comment I made.

**Pull request merged**